### ISSUE TITLE
Node provider doesn't work for MacOS and Windows when running on Docker

### ISSUE THREAD

**Issue by etherunit** [mysteriumnetwork/node#1599]
Attempted to connect to my myst node with id: 0x12ed33ad8ef814b3d391ac0ab1e5b393bd74cbff(0.20.0 launched on a Docker) via mobile app. Received the following error
`2020-01-30T12:28:27.471 ERR nat/traversal/pinger.go:285              > Failed to read remote peer: 88.119.191.24:45448 - attempting to continue error="read udp 172.17.0.2:46679->88.119.191.24:45448: read: connection refused"

2020-01-30T12:28:27.641 DBG nat/traversal/pinger.go:198              > Pinging... 

2020-01-30T12:28:27.671 ERR nat/traversal/pinger.go:285              > Failed to read remote peer: 88.119.191.24:45448 - attempting to continue error="read udp 172.17.0.2:46679->88.119.191.24:45448: read: connection refused"

2020-01-30T12:28:27.843 DBG nat/traversal/pinger.go:198              > Pinging... 

2020-01-30T12:28:27.876 ERR nat/traversal/pinger.go:285              > Failed to read remote peer: 88.119.191.24:45448 - attempting to continue error="read udp 172.17.0.2:46679->88.119.191.24:45448: read: connection refused"

2020-01-30T12:28:27.876 ERR nat/traversal/pinger.go:338              > Ping receiver error error="**NAT punch attempt timed out**"` 
App has disconneted. Tried from different devices.

**Comment by anjmao** [comment]
To be more clear. This issue appears when running docker on mac or windows. Since it doesn't allow to use OS host networking NAT hole punching fails. Remember that docker runs on separate VM on Mac and Windows.

For Macos users simple solution would be to simply run go compiled binary (no need to use docker at all)
```sh
 sudo myst --testnet service --agreed-terms-and-conditions  wireguard,openvpn
```

For Windows users it could be the same to run compiled binary but we don't support windows yet.

**Comment by anjmao** [comment]
As we discussed the best approach would be to allow user to specify range of ports which we can map and use for NAT pinging.

**Comment by soffokl** [comment]
This should be resolved with a 0.22 version and new TTL based NAT hole punching.
It is possible to run docker without host network and forwarded ports now.
